docs/src/development/proposals/MEP8/README.md

```diff
@@ -0,0 +1,363 @@
+```
+
+The `default` layout reflects what is actually implemented in metal-hammer to guarantee backward compatibility.
+
+```yaml
+---
+id: default
+constraints:
+  sizes:
+    - c1-large-x86
+    - c1-xlarge-x86
+  images:
+    - "debian*"
+    - "ubuntu*"
+    - "centos*"
+filesystems:
+  - path: "/boot/efi"
+    device: "/dev/sda1"
+    format: "vfat"
+    options: "-F 32"
+  - path: "/"
+    device: "/dev/sda2"
+    format: "ext4"
+  - path: "/var/lib"
+    device: "/dev/sda3"
+    format: "ext4"
+  - path: "/tmp"
+    device: "tmpfs"
+    format: "tmpfs"
+    mountoptions: ["defaults","noatime","nosuid","nodev","noexec","mode=1777","size=512M"]
+disks:
+  - device: "/dev/sda"
+    partitionprefix: "/dev/sda"
+    wipe: true
+    partitions:
+      - number: 1
+        label: "efi"
+        size: 500000000
+        guid: EFISystemPartition
+        type: GPTBoot
+      - number: 2
+        label: "root"
+        size: 5000000000
+        type: GPTLinux
+      - number: 3
+        label: "varlib"
+        size: 0 # to end of partition
+        type: GPTLinux
+```
+
+The `firewall` layout reuses the built in nvme disk to store the logs, which is way faster and larger than what the sata-dom ssd provides.
+
+```yaml
+---
+id: firewall
+constraints:
+  sizes:
+    - c1-large-x86
+    - c1-xlarge-x86
+  images:
+    - "firewall*"
+filesystems:
+  - path: "/boot/efi"
+    device: "/dev/sda1"
+    format: "vfat"
+    options: "-F 32"
+  - path: "/"
+    device: "/dev/sda2"
+    format: "ext4"
+  - path: "/var"
+    device: "/dev/nvme0n1p1"
+    format: "ext4"
+disks:
+  - device: "/dev/sda"
+    partitionprefix: "/dev/sda"
+    wipe: true
+    partitions:
+      - number: 1
+        label: "efi"
+        size: 500000000
+        guid: EFISystemPartition
+        type: GPTBoot
+      - number: 2
+        label: "root"
+        size: 5000000000
+        type: GPTLinux
+  - device: "/dev/nvme0n1"
+    partitionprefix: "/dev/nvme0n1p"
+    wipe: true
+    partitions:
+      - number: 1
+        label: "var"
+        size: 0
+        type: GPTLinux
+```
+
+The `storage` layout will be used for the storage servers, which must have mirrored boot disks.
+
+```yaml
+---
+id: storage
+constraints:
+  sizes:
+    - s3-large-x86
+  images:
+    - "centos*"
+filesystems:
+  - path: "/boot/efi"
+    device: "/dev/md1"
+    format: "vfat"
+    options: "-F32"
+  - path: "/"
+    device: "/dev/md2"
+    format: "ext4"
+disks:
+  - device: "/dev/sda"
+    partitionprefix: "/dev/sda"
+    wipe: true
+    partitions:
+      - number: 1
+        label: "efi"
+        size: 500000000
+        guid: EFISystemPartition
+        type: GPTLinuxRaid
+      - number: 2
+        label: "root"
+        size: 5000000000
+        type: GPTLinuxRaid
+  - device: "/dev/sdb"
+    partitionprefix: "/dev/sdb"
+    wipe: true
+    partitions:
+      - number: 1
+        label: "efi"
+        size: 500000000
+        guid: EFISystemPartition
+        type: GPTLinuxRaid
```

Comment:
The change of guid to EFISystemPartition set also the partition type to GPTBoot, so after a second set to type GPTLinuxRaid, i think the last change/set is the result. Like on my comment above it is not necessary to set the guid to  EFISystemPartition.
Generally I am not sure what's the right partition type for EFI-Boot-Partition in a Raid-Environment, EFISystemPartition or GPTLinuxRaid ?
On our test-server we use the partition type GPTLinuxRaid on both disk's for the EFI partition to build the Raid and it works, the server can boot from both disk's although the type is **not** set to EFISystemPartition !

Reply:
Thanks, also removed